### PR TITLE
Services coverage

### PR DESCRIPTION
**Pull request by ulitol97**
Better code coverage for the services, as part of #29

### CODE REVIEW

File: src/test/java/com/uniovi/main/services/AgentsServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	private AgentInfo testInfo1;
+	private AgentInfo testInfo2;
+
+
+	@After
+	public void clean() {
+		if (testInfo1 != null)
+			agentsService.deleteAgent(testInfo1);
+		if (testInfo2 != null)
+			agentsService.deleteAgent(testInfo1);
```
Comment (carmee-en) [inline src/test/java/com/uniovi/main/services/AgentsServiceTest.java]: This is why one of the tests fail. You never really delete testInfo2 after a test method.

**Comment by carmee-en** [comment]
**(Speculation here)** About InsertSampleDataServiceTest, could it be failing because the finalize method (which removes the incidents and agents created) in InsertSampleDataService has already been executed? Either that or the service never initialised.

The code seems okay to me, so I honestly don't understand.

**Comment by ulitol97** [comment]
Thanks for the advice. I think the error must be in that direction. Anyway, it was late night when I was trying to do that, so I tried some quick fixes (my bad) and decided to sleep a little. I'll think quietly about it today and should be done by the end of day ^^

**Comment by ulitol97** [comment]
I see the codecov/patch check complaining about what seems to be the InsertSampleDataService , despite this, which I don't understand at all, the service is used for testing and the project's codecov is increasing as expected, so I'd personally merge this.

**Comment by carmee-en** [comment]
From what I understand '_codecov/patch 25% of diff hit (target 68.7%)_' means that of the total lines you modified (i guess not including test files), only 25% are covered. I think that is mostly because the postConstruct in InsertSampleDataService, where some lines where added, is not tested. 

Don't worry about it though, the codecov increased and that is the most important.

**Pull request merged**